CoEpi/domain/model/KeyValueStore.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class KeyValueStore {
+    static let boolDefaults = ["showOnboarding": true]
+    
+    static func putBool(key: String, value: Bool) {
+        let defaultResult = boolDefaults[key]
+        if defaultResult == nil {
+            print("Error: Storing bool for invalid key: " + key)
+        } else {
+            UserDefaults.standard.set(value, forKey: key)
+        }
+    }
+    
+    static func getBool(key: String) -> Bool {
+        let defaultResult = boolDefaults[key]
+        if defaultResult == nil {
+            print("Error: Getting bool for invalid key: " + key)
+            return defaultResult! // Will throw an error
```

Comment:
Let's avoid force unwrap (`!`), unless it's really unavoidable. Here it's not (this code will disappear either way if you remove `boolDefaults`. Just sayin').

Reply:
Force unwrap is no longer used.